transformer_engine/pytorch/module/base.py

```diff
@@ -64,7 +64,7 @@ def get_workspace() -> torch.Tensor:
 
 def initialize_ub(
```

Comment:
Since `base.py` will now contain exposed API, maybe we should add an `__all__` like other modules directive.